src/functions/errorMessageGenerator.ts

```diff
@@ -11,14 +11,18 @@ export function errorMessageGenerator(
 ): ErrorMessageMessage {
   if (isLinkProvided(url)) {
     if (linkStartsWithHttps(url)) {
-      if (isLinkValid(url)) {
-        if (isLinkDuplicate(url, items)) {
-          return errorMessages.duplicate;
+      if (url.length >= 2083) {
```

Comment:
Same here with the constant = also a good example to reuse the constant definition from one place.

Say it changes in future to 2084 - then we only need to change the value in one place...